imports/ui/pages/Profile.tsx

```diff
@@ -83,42 +83,84 @@ export const Profile = () => {
 
   const profilePicture = watch("image");
 
+  const form = useForm();
+
   return (
-    <div>
-      <h1>Profile</h1>
-      <form onSubmit={handleSubmit(handleUpdateProfile)}>
-        {profilePicture && (
-          <div>
-            <img
-              src={profilePicture}
-              alt="Profile Picture"
-              style={{ width: "150px", height: "150px", borderRadius: "50%" }}
+    <div className="grid gap-4 grid-cols-2 grid-rows-4 max-w-4xl mx-auto p-8">
+      <div className="bg-white p-8 rounded-md shadow-sm">
+        <h1 className="text-2xl font-bold mb-4">Your Account</h1>
+        <p className="text-sm text-gray-500">Update your account's name and email address.</p>
+      </div>
+      <div className="bg-slate-100 p-8 rounded-md shadow-md">
+        <Form {...form}>
+          <form onSubmit={handleSubmit(handleUpdateProfile)} className="space-y-6">
+            <div className="flex items-center gap-4">
+              {profilePicture ? (
+                <img
+                  src={profilePicture}
+                  alt="Profile"
+                  className="w-32 h-32 rounded-full bg-gray-200 border"
+                />
+              ) : (
+                <div className="w-16 h-16 rounded-full bg-gray-200 flex items-center justify-center text-gray-400">
+                  H
```

Comment:
what happens if my user starts with Z? create a function that will handle this logic

Reply:
Nice catch, fixed already.
Thank you!